### DISCUSSION TITLE
Cannot deploy my next.js app to vercel

### DISCUSSION THREAD

**Discussion by Vette1123** [vercel/next.js#68809] [Help]
### Link to the code that reproduces this issue

https://github.com/Vette1123/next-js-real-time-chat

### To Reproduce

![image](https://github.com/user-attachments/assets/dcda4569-11a6-4cd5-8332-4356a66786ce)

I get this useless error when trying to deploy my project to vercel (using pusher for real time)

### Current vs. Expected behavior

![image](https://github.com/user-attachments/assets/dcda4569-11a6-4cd5-8332-4356a66786ce)

I get this useless error when trying to deploy my project to vercel (using pusher for real time)

### Provide environment information

```bash
![image](https://github.com/user-attachments/assets/dcda4569-11a6-4cd5-8332-4356a66786ce)

I get this useless error when trying to deploy my project to vercel (using pusher for real time)
```

### Which area(s) are affected? (Select all that apply)

create-next-app

### Which stage(s) are affected? (Select all that apply)

Vercel (Deployed)

**Comment by icyJoseph** [comment]
The repository shared is private.

Have you checked that this doesn't happen locally when running `next build` or `next start`?

Could you share more of the logs? I think this is a common issue, with an easy fix, but I'd like to see more of the error logs.

**Comment by Eliot6001** [comment]
As previously mentioned, The repository is private.

Have you attempted to run the command locally? 
```
turbo build
```  

Have you checked the basic setup, [env variables in turbo](https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#examples) and whether it is correct?
